mayastor/tests/nexus_multipath.rs

```diff
@@ -138,6 +140,51 @@ async fn nexus_multipath() {
         );
     }
 
+    let output_list = Command::new("nvme").args(&["list"]).output().unwrap();
+    assert!(
+        output_list.status.success(),
+        "failed to list nvme devices, {}",
+        output_list.status
+    );
+    let sl = String::from_utf8(output_list.stdout).unwrap();
+    let nvmems: Vec<&str> = sl
```

Comment:
nvme-cli does support JSON output BTW... not saying it makes life easier in this case but perhaps for future references.

Reply:
Could we use the `nvmeadm` library for this?

Reply:
I forgot that it does, the existing text-scraping in that test was for the `disconnect` command, which doesn't support output formats. The `nvmeadm` library doesn't (yet) support `ana_state` or possibly the concept of ANA. I'm leaning towards leaving the code here as-is as it's already written and leave that for future improvement.